src/common/enums/query_rel_type.cpp

```diff
@@ -0,0 +1,23 @@
+#include "common/enums/query_rel_type.h"
+#include "common/assert.h"
+
+namespace kuzu {
+namespace common {
+
+PathSemantic QueryRelTypeUtils::getPathSemantic(QueryRelType queryRelType) {
+    switch (queryRelType) {
+    case QueryRelType::VARIABLE_LENGTH_WALK:
+    case QueryRelType::SHORTEST:
```

Comment:
Shortest and All Shortest (unweighted) are by default both acyclic and trail though I'd mark them as trail. Why did you mark them as walk?

Reply:
They don't make much difference when we do the BFS backtrack because the graph itself doesn't contain duplicate nodes anyways. But I have changed semantic to acyclic as you requested.